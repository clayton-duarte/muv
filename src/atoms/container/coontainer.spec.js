import React from 'react';

import Container from '.';

it('should render a Container', () => {
  const wrapper = mount(
    <Container />,
  );
  expect(wrapper).toMatchSnapshot();
});
