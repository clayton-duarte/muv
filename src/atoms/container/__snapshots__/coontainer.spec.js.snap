// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a Container 1`] = `
<Container
  full={false}
  height="auto"
  margin="0"
  maxHeight="auto"
  maxWidth="auto"
  minHeight="auto"
  minWidth="auto"
  padding="0"
>
  <styled.div
    full={false}
    height="auto"
    margin="0"
    maxHeight="auto"
    maxWidth="auto"
    minHeight="auto"
    minWidth="auto"
    padding="0"
    theme={
      Object {
        "color": Object {
          "black": "#222222",
          "darkgray": "#515151",
          "error": "#dd3333",
          "gray": "#a9abae",
          "lightgray": "#e5e5e5",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0081c8",
          "secondary": "#00245f",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "font": Object {
          "color": "#222222",
          "family": "\\"Ubuntu\\", sans-serif",
          "size": "16px",
          "spacing": 0,
        },
        "shape": Object {
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".75rem .875rem",
          "radius": ".5rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
          "size": "16px",
        },
        "transition": Object {
          "time": ".3s ease",
        },
      }
    }
  >
    <StyledComponent
      forwardedClass={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": false,
            "lastClassName": "bofdhh",
            "rules": Array [
              "
padding: ",
              [Function],
              ";
flex-basis: ",
              [Function],
              ";
border: ",
              [Function],
              ";
display: ",
              [Function],
              ";
height: ",
              [Function],
              ";
width: ",
              [Function],
              ";
background: ",
              [Function],
              ";
border-radius: ",
              [Function],
              ";
max-height: ",
              [Function],
              ";
min-height: ",
              [Function],
              ";
max-width: ",
              [Function],
              ";
min-width: ",
              [Function],
              ";
margin: ",
              [Function],
              ";
flex-direction: column;
align-items: stretch;
",
            ],
          },
          "defaultProps": Object {
            "theme": Object {
              "color": Object {
                "black": "#222222",
                "darkgray": "#515151",
                "error": "#dd3333",
                "gray": "#a9abae",
                "lightgray": "#e5e5e5",
                "overlay": "rgba(0, 0, 0, .15)",
                "primary": "#0081c8",
                "secondary": "#00245f",
                "success": "#33bb33",
                "warn": "#dd3333",
                "white": "#fefefe",
              },
              "font": Object {
                "color": "#222222",
                "family": "\\"Ubuntu\\", sans-serif",
                "size": "16px",
                "spacing": 0,
              },
              "shape": Object {
                "border": ".125rem solid rgba(0, 0, 0, .15)",
                "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
                "margin": ".25rem .125rem",
                "padding": ".75rem .875rem",
                "radius": ".5rem",
                "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                "size": "16px",
              },
              "transition": Object {
                "time": ".3s ease",
              },
            },
          },
          "displayName": "styled.div",
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
      full={false}
      height="auto"
      margin="0"
      maxHeight="auto"
      maxWidth="auto"
      minHeight="auto"
      minWidth="auto"
      padding="0"
      theme={
        Object {
          "color": Object {
            "black": "#222222",
            "darkgray": "#515151",
            "error": "#dd3333",
            "gray": "#a9abae",
            "lightgray": "#e5e5e5",
            "overlay": "rgba(0, 0, 0, .15)",
            "primary": "#0081c8",
            "secondary": "#00245f",
            "success": "#33bb33",
            "warn": "#dd3333",
            "white": "#fefefe",
          },
          "font": Object {
            "color": "#222222",
            "family": "\\"Ubuntu\\", sans-serif",
            "size": "16px",
            "spacing": 0,
          },
          "shape": Object {
            "border": ".125rem solid rgba(0, 0, 0, .15)",
            "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
            "margin": ".25rem .125rem",
            "padding": ".75rem .875rem",
            "radius": ".5rem",
            "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
            "size": "16px",
          },
          "transition": Object {
            "time": ".3s ease",
          },
        }
      }
    >
      <div
        className="sc-bdVaJa bofdhh"
        height="auto"
      >
        container
      </div>
    </StyledComponent>
  </styled.div>
</Container>
`;
